packages/fast-components-react-base/src/auto-suggest/README.md

```diff
@@ -10,5 +10,7 @@ The expected usage pattern of the component is for authors to watch the "onValue
 
 The input region of the component can be customized using the component's "inputRegion" render prop.
 
+The "isMenuOpen" prop enables authors to take control of when the menu is open.  Setting "isMenuOpen" to false keeps the menu closed, and setting it to true keeps it open (unless there are no children, in which case it is always closed).
```

Comment:
I think you can remove this since it is not a prop but internal state.

Reply:
It is actually both.  The prop makes the menu state controllable.